src/permission-ng/permission-ng.js

```diff
@@ -85,8 +85,8 @@
     });
   }
 
-  angular
+  module.exports = angular
     .module('permission.ng', ['permission', 'ngRoute'])
-    .run(run);
+    .run(run).name;
 
-}());
+}(module || {}));
```

Comment:
module will be undefined here. mb define it above like `var module = module || {};` ?